src/main/java/org/broadinstitute/hellbender/tools/gvs/common/SchemaUtils.java

```diff
@@ -83,6 +83,17 @@ public static long encodeLocation(String chrom, int position) {
         return (long) chromosomeIndex * chromAdjustment + (long) position;
     }
 
+
+    public static long encodeCompressedRefBlock(String chrom, long position, long length, long stateAsInt) {
+        long chromosomeIndex = ChromosomeEnum.valueOfContig(chrom).index;
+        long packedBytes = ((0xFFFF & chromosomeIndex) << 48) |
+                ((0xFFFFFFFF & position) << 16) |
+                ((0xFFF & length) << 4) |
+                (0xF & stateAsInt);
+
+        return packedBytes;
+    }
+
```

Comment:
There are a lot of "not covered by tests" warnings on the logic above that looks like it really could / should be covered by unit tests.

Reply:
Added a unit testes for the new code I put in SchemaUtils